osmaxx/conversion/converters/converter_garmin/garmin.py

```diff
@@ -18,18 +20,25 @@ def __init__(self, *, out_zip_file_path, area_name, polyfile_string):
         self._osmosis_polygon_file.write(polyfile_string)
         self._osmosis_polygon_file.flush()
         self._polyfile_path = self._osmosis_polygon_file.name
+        self._start_time = None
 
     def create_garmin_export(self):
-        self._to_garmin()
+        self._start_time = timezone.now()
+        unzipped_result_size = self._to_garmin()
```

Comment:
While it was not completely baffling (albeit maybe not really intuitive, either) that `._to_garmin()` returned the path to the resulting ZIP file, returning the content's cumulated file size is rather surprising.

As each export to GARMIN format should (and does) use a new `Garmin` instance (otherwise `._start_time` would lose meaning), `_produce_garmin()`, where the size is computed, could store the size on another instance variable instead of returning it.